src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,813 @@
+      {{#> content}}
+        <p>Support content</p>
+      {{/content}}
+    {{/tab-content}}
+  {{/page-main-section}}
+{{/tabs--page-wrapper}}
+```
+
+### Vertical tabs
+```hbs isFullscreen
+{{#> tabs--page-wrapper tabs--page-wrapper--id="vertical-tabs-example"}}
+  {{> page-template-breadcrumb page-template-breadcrumbs--IsOS="true"}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" page-main-section--modifier="pf-m-light"}}
+    {{#> title title--modifier="pf-m-2xl pf-u-mt-md"}}
+      Install OpenShift on AWS
+    {{/title}}
+  {{/page-main-section}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" tabs--id=(concat tabs--page-wrapper--id '-tabs') tab-content--id=(concat tabs--page-wrapper--id '-tabs')}}
+    {{#> card card--modifier="pf-m-fill"}}
+      {{#> sidebar}}
+        {{#> sidebar-panel}}
+          {{#> tabs tabs--modifier="pf-m-inset-md pf-m-inset-lg-on-xl pf-m-vertical pf-m-box" }}
+            {{#> tabs-list}}
+              {{> __tabs-item
+                __tabs-item--current="true"
+                __tabs-item--id="installation"
+                __tabs-item--aria-label="Installation"
+                __tabs-item--text="Installation"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-installation-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="limits"
+                __tabs-item--aria-label="Limits"
+                __tabs-item--text="Limits"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-limits-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="cluster-installation"
+                __tabs-item--aria-label="Cluster installation"
+                __tabs-item--text="Cluster installation"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-cluster-installation-panel"')}}
+              {{> __tabs-item
+                __tabs-item--id="subscription-support"
+                __tabs-item--aria-label="Subscription and support"
+                __tabs-item--text="Subscription and support"
+                __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-subscription-support-panel"')}}
+            {{/tabs-list}}
+          {{/tabs}}
+        {{/sidebar-panel}}
+        {{#> sidebar-content sidebar-content--modifier="pf-u-p-lg"}}
+          {{#> tab-content tab-content--IsActive="true" tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-installation-link" id="' tab-content--id '-installation-panel"')}}
+            {{#> content}}
+              <h2>Installation content</h2>
+              <p>This document is a guide for preparing a new AWS account for use with OpenShift. It will help prepare an account tot crea a single cluster and provide insight for adjustments which may be needed for additional cluster.</p>
```

Comment:
```suggestion
              <p>This document is a guide for preparing a new AWS account for use with OpenShift. It will help prepare an account to create a single cluster and provide insight for adjustments which may be needed for additional cluster.</p>
```